enhancements/network/allow-mtu-and-overlay-port-changes.md

```diff
@@ -0,0 +1,185 @@
+---
+title: allow-mtu-and-overlay-port-changes
+authors:
+  - "@juanluisvaladas"
+reviewers:
+  - "@danwinship"
+  - "@mccv1r0"
+approvers:
+  - TBD
+creation-date: 2021-01-25
+last-updated: 2022-02-03
+status: provisional
+---
+
+# Allow MTU and overlay port changes
+
+This covers adding the capability to the cluster network operator of changing
+the MTU, and the VXLAN and Geneve ports post installation.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Customers may need to change the MTU, or the ports used for VXLAN or Geneve
+tunnels post-installation. However these changes aren't trivial and may cause
+downtime, hence the CNO forbids currently them.
+
+We propose a brand new daemonset that will be launched on demand and will
+run on every node of the cluster. This daemonset will make the necessary
+changes in an ordered, and coordinated manner with a service disruption
+within the TCP timeout (best effort).
+
+## Motivation
+
+While customers usually set the MTU correctly, and the overlay ports some
+times they need to change them afterwards for unpredictable reasons such as
+changes in the underlay or because they set it to a wrong value when they
+installed the cluster.
+
+## Goals
+
+* Allow to change MTU of the tunnels post install on both OpenShift SDN and
+  OVN Kubernetes.
+
+* Allow to change the overlay network ports on the underlay in both OpenShift
+  SDN and OVN Kubernetes.
+
+* Allow to change both VXLAN and Geneve ports in a single operation.
```

Comment:
The wording of this goal is a bit confusing.  It sounds like both Geneve and VXLAN are used at the same time?

Reply:
My understanding is that they can, for hybrid overlay